classes/components/forms/statistics/users/ReportForm.inc.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * @file classes/components/form/statistics/users/ReportForm.inc.php
+ *
+ * Copyright (c) 2014-2020 Simon Fraser University
+ * Copyright (c) 2000-2020 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class AssignToIssueForm
+ * @ingroup classes_controllers_form
+ *
+ * @brief A preset form for configuring the users report.
+ */
+namespace PKP\components\forms\statistics\users;
+use \PKP\components\forms\{FormComponent, FieldOptions};
+
+class ReportForm extends FormComponent {
+	/**
+	 * Constructor
+	 *
+	 * @param string $action URL to submit the form to
+	 * @param \Context $context The context
+	 */
+	public function __construct(string $action, \Context $context)
+	{
+		$this->action = $action;
+		$this->id = 'reportForm';
+		$this->method = 'POST';
+
+		$this->addPage(array('id' => 'default', 'submitButton' => array('label' => __('common.export'))));
```

Comment:
We're gradually shifting from `array(...)` syntax to `[...]` -- feel free to use the more concise syntax for new code!

Reply:
This one escaped from a ctrl+c/v :D